flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesTestBase.java

```diff
@@ -18,102 +18,70 @@
 import org.apache.flink.configuration.Configuration;
-import org.apache.flink.configuration.ResourceManagerOptions;
-import org.apache.flink.configuration.TaskManagerOptions;
 import org.apache.flink.kubernetes.configuration.KubernetesConfigOptions;
-import org.apache.flink.kubernetes.configuration.KubernetesConfigOptionsInternal;
 import org.apache.flink.kubernetes.kubeclient.Fabric8FlinkKubeClient;
 import org.apache.flink.kubernetes.kubeclient.FlinkKubeClient;
-import org.apache.flink.kubernetes.kubeclient.decorators.Decorator;
-import org.apache.flink.kubernetes.kubeclient.decorators.InitializerDecorator;
-import org.apache.flink.kubernetes.kubeclient.decorators.OwnerReferenceDecorator;
-import org.apache.flink.kubernetes.kubeclient.decorators.ServiceDecorator;
-import org.apache.flink.kubernetes.kubeclient.resources.KubernetesService;
 import org.apache.flink.kubernetes.utils.Constants;
 import org.apache.flink.runtime.clusterframework.BootstrapTools;
 import org.apache.flink.test.util.TestBaseUtils;
 import org.apache.flink.util.TestLogger;
 
-import io.fabric8.kubernetes.api.model.LoadBalancerIngress;
-import io.fabric8.kubernetes.api.model.LoadBalancerStatus;
-import io.fabric8.kubernetes.api.model.Service;
-import io.fabric8.kubernetes.api.model.ServiceStatusBuilder;
-import io.fabric8.kubernetes.api.model.WatchEvent;
 import io.fabric8.kubernetes.client.KubernetesClient;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.rules.TemporaryFolder;
 
-import javax.annotation.Nullable;
-
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 /**
  * Base test class for Kubernetes.
  */
 public class KubernetesTestBase extends TestLogger {
-	@Rule
-	public MixedKubernetesServer server = new MixedKubernetesServer(true, true);
-
-	@Rule
-	public TemporaryFolder temporaryFolder = new TemporaryFolder();
-
-	private File flinkConfDir;
 
 	protected static final String NAMESPACE = "test";
-
-	protected static final Configuration FLINK_CONFIG = new Configuration();
-
 	protected static final String CLUSTER_ID = "my-flink-cluster1";
-
 	protected static final String CONTAINER_IMAGE = "flink-k8s-test:latest";
+	protected static final String CONTAINER_IMAGE_PULL_POLICY = "IfNotPresent";
 
-	protected static final String MOCK_SERVICE_ID = "mock-uuid-of-service";
+	@Rule
+	public MixedKubernetesServer server = new MixedKubernetesServer(true, true);
 
-	protected static final String MOCK_SERVICE_HOST_NAME = "mock-host-name-of-service";
+	@Rule
+	public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+	protected File flinkConfDir;
 
-	protected static final String MOCK_SERVICE_IP = "192.168.0.1";
+	protected final Configuration flinkConfig = new Configuration();
 
-	protected static final String FLINK_MASTER_ENV_KEY = "LD_LIBRARY_PATH";
+	protected KubernetesClient kubeClient;
```

Comment:
Since we make `kubeClient` and `flinkKubeClient` as `protected`, these methods `getFabric8FlinkKubeClient(), getKubeClient()` could be removed now.

Reply:
Fixed by [d78c1b1](https://github.com/apache/flink/pull/11233/commits/d78c1b1b282a3d7fcfc797bf909dca2b5f2264e4)